pkg/storegateway/partitioner.go

```diff
@@ -17,22 +17,23 @@ type gapBasedPartitioner struct {
 	maxGapBytes uint64
 
 	// Metrics.
-	requestedBytes  prometheus.Counter
-	requestedRanges prometheus.Counter
-	expandedBytes   prometheus.Counter
-	expandedRanges  prometheus.Counter
+	requestedBytes       prometheus.Counter
+	requestedRanges      prometheus.Counter
+	expandedBytes        prometheus.Counter
+	expandedRanges       prometheus.Counter
+	extendedRangesRanges prometheus.Counter
 }
 
 func newGapBasedPartitioner(maxGapBytes uint64, reg prometheus.Registerer) *gapBasedPartitioner {
 	return &gapBasedPartitioner{
 		maxGapBytes: maxGapBytes,
 		requestedBytes: promauto.With(reg).NewCounter(prometheus.CounterOpts{
 			Name: "cortex_bucket_store_partitioner_requested_bytes_total",
-			Help: "Total size of byte ranges required to fetch from the storage before they are passed to the partitioner.",
+			Help: "Total size of byte ranges required to fetch from the storage before they are extended with maxGapBytes.",
```

Comment:
`maxGapBytes` is the name of the internal variable, but not some user facing.

```suggestion
			Help: "Total size of byte ranges required to fetch from the storage before they are extended by the partitioner.",
```